### ISSUE TITLE
nasnet_large pretrain model key not found error

### ISSUE THREAD

**Issue by chenghuige** [tensorflow/models#2656]
tensorflow 1.4.rc1
Many keys not found like:
Not found: Key cell_17/comb_iter_1/right/bn_sep_3x3_1/gamma not found in checkpoin
Key cell_3/comb_iter_1/right/separable_3x3_2/pointwise_weights not found in checkpoint

For example it complains:
cell_2/comb_iter_0/left/separable_5x5_1/depthwise_weights not found in checkpoint
And in the checkpoint there is 
cell_2/comb_iter_0/left/separable_5x5_1/depthwise_weights/ExponentialMovingAverage (DT_FLOAT) [5,5,168,1]

I'm using code like blelow it worked for inceptionV3,V4,resnet_v2_152.. but nasnet_large with pretrain model fail
    net_fn = nets_factory.get_network_fn(net_name, num_classes=num_classes, 
                                                               is_training=is_image_model_training)
    logits, end_points = net_fn(image)

Another thing why not give nasnet a global prefix scope like nasnet_large or NasnetLarge just like inception and resnet models. It will make finetune later easier, now I have to manually add and convert the model.

**Comment by chenghuige** [comment]
Might due to moving_average_decay, close it.
